Source/WebCore/rendering/RenderLayerBacking.cpp

```diff
@@ -3892,6 +3896,8 @@ static RefPtr<Pattern> patternForEventListenerRegionType(EventListenerRegionType
             return { "sync"_s, { 0, 9 }, SRGBA<uint8_t> { 200, 0, 0, 128 } };
         case EventListenerRegionType::MouseClick:
             break;
+        default:
```

Comment:
I think you should do something for the overlay here in this patch.

Reply:
Added. Will need a separate patch to actually hook it up though.